### PR TITLE
Add a condition to set the header

### PR DESCRIPTION
**Pull request by exeto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,13 @@ app.use(hpkp({
   maxAge: ninetyDaysInMilliseconds,
   sha256s: ['AbCdEf123=', 'ZyXwVu456='],
   includeSubdomains: true,         // optional
-  reportUri: 'http://example.com'  // optional
-  reportOnly: false                // optional
+  reportUri: 'http://example.com', // optional
```
Comment (EvanHahn) [inline README.md]: Good catch here!

File: index.js [hunk 2]
```diff
@@ -18,9 +28,13 @@ function parseOptions (options) {
 
   var maxAge = options.maxAge
   var sha256s = options.sha256s
+  var setIf = options.setIf
 
   if (!maxAge || maxAge <= 0) { throw badArgumentsError }
   if (!sha256s || sha256s.length < 2) { throw badArgumentsError }
+  if (setIf && !(typeof setIf === 'function')) {
```
Comment (EvanHahn) [inline index.js]: Could you change this line to:

```js
if (setIf && (typeof setIf !== 'function')) {
```

**Comment by EvanHahn** [comment]
I like this! Could you add tests and I'll merge?

**Comment by exeto** [comment]
@EvanHahn done.

**Comment by EvanHahn** [comment]
Perfect! Thanks so much.

If you'd like to add yourself in the `contributors` part of `package.json`, feel free. I'll merge and release `hpkp@1.1.0` tomorrow!

**Comment by EvanHahn** [comment]
This is released in `hpkp@1.1.0` and `helmet@1.3.0`. Thanks!

**Comment by exeto** [comment]
Thanks

**Comment by EvanHahn** [comment]
I'm building a list of contributors for a new Helmet.js website. Do you have a name and/or website you want me to use to credit you?

**Comment by exeto** [comment]
Timofey Dergachev, https://exeto.me/en

**Comment by EvanHahn** [comment]
Thanks! Added you to the list—stay tuned for the full website.

**Comment by exeto** [comment]
Thanks

**Pull request merged**